src/api/routes/accessRequest.ts

```diff
@@ -0,0 +1,27 @@
+import { Request, Response } from 'express';
+import AsyncRouter from '../AsyncRouter';
+import { requestAccess } from '../../application/service';
+import { User } from '../../domain/model/accessRequest/AccessRequest';
+
+export default () => {
+  const route = new AsyncRouter();
+
+  route.post(
+    '/users/:id/access-requests',
```

Comment:
It seems like here we are posting an `access-request` to a different userId that doesn't belong to me. Given we already have an approval (sharingCode) this doesn't seem right to me.

I would instead do the following
```
POST /users/:id/access-passes
{ sharingCode: <UUID> }

Where `id` is the user's ID. This way we can also have a GET request that can give us a list of all the users to whom we still have an active pass.

GET /users/:id/access-passes
[ { userId: 10 }. { userId: 20 }, ...]
```

Then we can also implement access-requests (again under the same user's ID) that would not necessarily need a user's presense. Instead we can send a notification to the other user. (Not sure if that will ever be needed)